### PR TITLE
Replace walletContents cookie storage with localStorage

### PR DESCRIPTION
**Pull request by nidnogg**
Proposal for fixing issue https://github.com/credential-handler/chapi-demo-wallet/issues/18

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Using `localStorage` wasn't possible in the past because of inaccessibility in Safari even with the Storage Access API -- but we don't use that API anymore and use 1st party storage in Safari (and soon to be all browsers). Therefore I think this probably works fine now.

We'll want to run tests on this across various browsers.

**Comment by nidnogg** [comment]
The only other place where cookies were used was for user storage - perhaps a follow up commit on this PR to replace that case as well and trim that cookie lib dependency out?

**Comment by dlongley** [comment]
@nidnogg,

I'm *pretty sure* we don't need to use cookies anywhere anymore, so +1 for removing that cookie lib dependency if it turns out all the browsers work with the `localStorage` approach for everything we need to store.

**Comment by nidnogg** [comment]
Added on latest commit, also tested on latest versions of Firefox, Chrome and Safari

File: mock-user-management.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function storeInWallet(verifiablePresentation) {
 
   // base64 encode the serialized contents (verifiable presentations)
   const serialized = btoa(JSON.stringify(walletContents));
-  Cookies.set('walletContents', serialized, {path: '', secure: true, sameSite: 'None'});
+  localStorage.setItem('walletContents', serialized);
```
Comment (dlongley) [inline mock-user-management.js:80]: I'm pretty sure we could just use JSON now as well. That would make the data a bit more readable / easier to debug when using devtools to read local storage directly. IOW, we shouldn't need to base64-encode the data to store in `localStorage`, that was only needed to store as a cookie.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: mock-user-management.js (hunk 1 above)
Comment (nidnogg) [reply mock-user-management.js]: Added a follow up commit to handle this

**Review by dlongley — APPROVED** [review]

**Pull request merged**